packs/elastic/actions/alias.yaml

```diff
@@ -0,0 +1,88 @@
+---
+description: Index Aliasing
+enabled: true
+entry_point: curator_runner.py
+name: alias
+parameters:
+  action:
+    default: alias
+    immutable: true
+    type: string
+  all_indices:
+    description: Do not filter indices.  Act on all indices.
+    type: boolean
+  dry_run:
+    default: false
+    description: Do not perform any changes.
+    type: boolean
+  exclude:
+    description: Index list to exclude from the operation.
+    items:
+      type: string
+    type: array
+  host:
+    description: Elasticsearch host.
+    required: true
+    type: string
+  http_auth:
```

Comment:
I would split this out to `username/password`, if only because we are working on backend schema changes to prevent leakage of secrets. If there is a munged text field here, we'll have leakage for sure.